crates/cli/src/subcommands/identity.rs

```diff
@@ -84,10 +78,7 @@ fn get_subcommands() -> Vec<Command> {
             ),
         Command::new("list").about("List saved identities which apply to a server")
             .arg(
-                Arg::new("server")
-                    .short('s')
-                    .long("server")
-                    .help("The nickname, host name or URL of the server to list identities for")
```

Comment:
note that this helptext changed by using the `common_args` version. I don't personally feel that this "custom" version is meaningfully more helpful.